src/Docker/JobScopedEncryptor.php

```diff
@@ -28,10 +27,21 @@ public function __construct(ObjectEncryptor $encryptor, string $componentId, str
     public function encrypt($data)
     {
         if ($this->configId === null) {
-            return $this->encryptor->encryptForProject($data, $this->componentId, $this->projectId);
+            return $this->encryptor->encryptForBranchType(
```

Comment:
Tim padem ale, pokud se stane, ze neco encryptuje tou stavajici variantou (bez branch), pak se tohle releasne a pokusi se to dektruptovat novou variantou (s branch), tak to vyfailuje, ne?

Reply:
je to v pořádku, dekryptovat novou variantou https://github.com/keboola/object-encryptor/blob/main/tests/ObjectEncryptorTest.php#L1021